Code/GraphMol/ForceFieldHelpers/ANI/ANI.h

```diff
@@ -0,0 +1,56 @@
+#include <RDGeneral/export.h>
+#ifndef RD_ANICONVENIENCE_H_
+#define RD_ANICONVENIENCE_H_
+
+#include <ForceField/ForceField.h>
+#include <GraphMol/ForceFieldHelpers/FFConvenience.h>
+#include "Builder.h"
+
+namespace RDKit {
+class ROMol;
+namespace ANI {
+//! Convenience function for optimizing a molecule using ANI
+/*
+  \param mol          the molecule to use
+  \param maxIters     the maximum number of force-field iterations
+  \param model        Param dir, such as one in /Code/ForceField/ANI/Params/
+  \param confId       the optional conformer id, if this isn't provided, the
+  molecule's default confId will be used.
+
+  \return a pair with:
+     first: 0 if the optimization converged, 1 if more iterations are required.
```

Comment:
Maybe `converged` should be a `bool`, then, if there are only two allowed values?

    std::pair<bool, double> ANIOptimizeMolecule(ROMol &mol, std::string model,
                                           int confId = -1,
                                           int maxIters = 1000);

Call like (C++17):

    auto [converged, energy] =  ANIOptimizeMolecule(mol, "ANI-1x");

Reply:
I agree in principle, but, pair<int, double> pattern is how the analogous methods work for the other force fields. See std::pair<int, double> MMFFOptimizeMolecule in ForceFieldHelpers/MMFF/MMFF.h, and std::pair<int, double> UFFOptimizeMolecule in ForceFieldHelpers/UFF/UFF.h